lms/templates/login.html

```diff
@@ -162,6 +159,15 @@ <h3 class="message-title">${_("We couldn't log you in.")} </h3>
         <p class="instructions"> </p>
       </div>
 
+      % if third_party_auth_error:
+        <div role="alert" class="status message third-party-auth-error is-shown" tabindex="-1">
+          <h3 class="message-title">${_("We're sorry, that didn't work.")} </h3>
```

Comment:
Is this ok? The new text may be a bit long for the UI:

![screen shot 2015-06-11 at 5 27 52 pm](https://cloud.githubusercontent.com/assets/945577/8121332/388c96ee-105f-11e5-92ba-47397e176b5d.png)